Duplicate the SQLITE3 issue when called from the Python Framework in Domoticz
